### PR TITLE
adding homework#13

### PR DESCRIPTION
**Pull request by ValentynaVel**

### CODE REVIEW

**Review by klymenok — CHANGES_REQUESTED** [review]
There is no solutions for home work

**Comment by ValentynaVel** [comment]
I just updated PR because files was empty.

File: homework13_3.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    # 120  | 2
+    # 1142 | 1
+
+import sqlite3
+from pprint import pprint
+
+conn = sqlite3.connect('database.sqlite')
+cursor = conn.cursor()
+
+query = ("SELECT age, COUNT(id) FROM users GROUP BY age")
```
Comment (klymenok) [inline homework13_3.py]: To have the output as it asked in the task, you should use alias here for the aggregated amount: ` ... , COUNT(id) as users FROM ...`

File: homework13_4.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    # 32    | 1
+    # 1142 | 1
+
+import sqlite3
+from pprint import pprint
+
+conn = sqlite3.connect('database.sqlite')
+cursor = conn.cursor()
+
+query = ("SELECT age, COUNT(id) FROM users GROUP BY age ORDER BY COUNT(id), age desc")
```
Comment (klymenok) [inline homework13_4.py]: You can use alias here for COUNT(id) and order by the alias: `... COUNT(id) as users ... ORDER BY users.

And pay attention: you should order by users in descending order and by age in ascending

**Review by klymenok — CHANGES_REQUESTED** [review]

**Review by klymenok — APPROVED** [review]
Looks good!

**Pull request merged**